L10nSnapshotTests/L10nSnapshotTests.swift

```diff
@@ -153,39 +153,28 @@ class L10nSnapshotTests: L10nBaseSnapshotTests {
         snapshot("13ContextMenuReloadButton-03", waitForLoadingIndicator: false)
     }
 
-    func test16PasscodeSettings() {
-        navigator.goto(SetPasscodeScreen)
-        snapshot("16SetPasscodeScreen-1-nopasscode")
-        userState.newPasscode = "111111"
-        navigator.performAction(Action.SetPasscodeTypeOnce)
-        snapshot("16SetPasscodeScreen-2-typepasscode")
-
-        userState.newPasscode = "111112"
-        navigator.performAction(Action.SetPasscodeTypeOnce)
-        snapshot("16SetPasscodeScreen-3-passcodesmustmatch")
-
-        userState.newPasscode = "111111"
-        navigator.performAction(Action.SetPasscode)
-        snapshot("16SetPasscodeScreen-3")
-
-        navigator.goto(PasscodeIntervalSettings)
-        snapshot("16PasscodeIntervalScreen-1")
-
-    }
-
-    func test17PasswordSnackbar() {
-        navigator.openURL("\(testPageBase)/password.html")
-        app.webViews.element(boundBy: 0).buttons["submit"].tap()
-        snapshot("17PasswordSnackbar-01")
-        app.buttons["SaveLoginPrompt.saveLoginButton"].tap()
-        // The password is pre-filled with a random value so second this this will cause the update prompt
-        navigator.openURL("\(testPageBase)/password.html")
-        app.webViews.element(boundBy: 0).buttons["submit"].tap()
-        snapshot("17PasswordSnackbar-02")
-    }
+//    func test16PasscodeSettings() {
```

Comment:
I got this test working as it is after restarting the simulator. It happens sometimes with these Authentication tests that the keyboard is not shown and so the test fails. Please try that and if it works also for you it could be uncommented.